src/server/pkg/deploy/assets/assets.go

```diff
@@ -411,7 +427,8 @@ func RethinkRc(backend backend, volume string, hostPath string) *api.Replication
 		spec.Spec.Template.Spec.Volumes[0].PersistentVolumeClaim = &api.PersistentVolumeClaimVolumeSource{
 			ClaimName: rethinkVolumeClaimName,
 		}
-	} else if backend == localBackend {
+	} else if backend == localBackend || backend == microsoftBackend {
+		// ToDo: workaround until https://github.com/pachyderm/pachyderm/issues/960
```

Comment:
Little confused by this comment, #960 is the Azure Support issue so it seems like this PR is kinda dependent on itself. Wasn't there something about needing k8s 1.4 to do provide data disks to rethink/etcd? Is that what you're referencing here?